sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/ImplementIDisposableCorrectlyPartial1.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public partial class ImplementIDisposableCorrectlyPartial : IDisposable // Noncompliant
+    {
+        protected virtual void Dispose(bool disposing)
+        {
+        }
+    }
+}
+
+
```

Comment:
Ahahah it is linked to the way we report on partial classes. It reports everything on the first class so I have put some extra spaces before to really make it clear this is the line corresponding to the other partial.